csrc/index_compute.cpp

```diff
@@ -57,9 +57,6 @@ int getProducerHaloOffset(
   // producer tensors, where reduction axes are skipped, producer_id
   // should never be a reduction axis.
   if (it == p2c.end()) {
-    TORCH_WARN(
-        "getProducerHaloOffset p2c mapping has failed. See "
-        "https://github.com/NVIDIA/Fuser/issues/1122");
```

Comment:
Is this unrelated?

Reply:
I suppose yes. @zasdfgbnm can you please leave a comment?

Reply:
This warning is issued in every matmul kernel, which is very annoying. The fundamental problem for this is because our halo analysis assumes we are indexing into rFactor domain, and when rFactor domain mismatch with allocation domain, the halo analysis will just fail. However, halo is not something we prioritize right now, so I am just doing whatever easiest hack to unblock, as long as in practice the code works. We need to rewrite indexing and abandon this code anyway.

Reply:
Yes, it'll be rewritten, and for that I'd like to keep track of what's working and what isn't. A code comment like you just mentioned would be greatly helpful for when we would revisit here.

Reply:
Oh I see what you mean. I will include some description in my next PR.

Reply:
description added in https://github.com/NVIDIA/Fuser/pull/1176